source/constructs/lib/live-streaming.ts

```diff
@@ -11,20 +11,21 @@
  *  and limitations under the License.                                                                                *
  *********************************************************************************************************************/
 
-import * as cdk from '@aws-cdk/core';
-import * as iam from '@aws-cdk/aws-iam';
-import * as lambda from '@aws-cdk/aws-lambda';
-import * as s3 from '@aws-cdk/aws-s3';
-import { Secret } from '@aws-cdk/aws-secretsmanager';
-import * as cloudfront from '@aws-cdk/aws-cloudfront';
-import * as origin from '@aws-cdk/aws-cloudfront-origins';
-import * as appreg from '@aws-cdk/aws-servicecatalogappregistry';
-import * as applicationinsights from '@aws-cdk/aws-applicationinsights';
+import * as cdk from 'aws-cdk-lib';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
+import * as cloudfront from 'aws-cdk-lib/aws-cloudfront';
+import * as origin from 'aws-cdk-lib/aws-cloudfront-origins';
+import * as appreg from '@aws-cdk/aws-servicecatalogappregistry-alpha';
+import * as applicationinsights from 'aws-cdk-lib/aws-applicationinsights';
```

Comment:
Remove applicationinsights. No longer required at deployment, IG has instructions for customers to enable it if they want